pageserver/src/walreceiver.rs

```diff
@@ -27,56 +27,54 @@ use crate::config::PageServerConf;
 use crate::task_mgr::WALRECEIVER_RUNTIME;
```

Comment:
NIT: ~4 more etcd words are in storage, on top of this file is one of them

> keys in etcd broker

Reply:
removed